README.md

```diff
@@ -854,6 +855,139 @@ registration: {
 ```
 - **registration.click** - Function that is called when the registration button is clicked
 
+
+## Registration
+
+```javascript
+    var signIn = new OktaSignIn({
+      baseUrl: 'https://acme.okta.com',
+      clientId: '{{myClientId}}',
+      registration: {
+        parseSchema: function(schema, onSuccess, onFailure) {
+           // handle parseSchema callback
+        },
+        preSubmit: function (postData, onSuccess, onFailure) {
+           // handle preSubmit callback
+        },
+        postSubmit: function (response, onSuccess, onFailure) {
+            // handle postsubmit callback
+        }
+      },
+      features: {
+         registration: true
+      }
+    });
+```
+
+Following configurations are needed to enable registration on a self hosted sign in widget
+
+
+- **features.registration:** Used to enable registration feature on the widget.
+    ```javascript
+    // https://github.com/okta/okta-signin-widget#feature-flags
+    features.registration: 'true'
+
+    ```
+- **clientId:** Client Id pre-registered with Okta for the OIDC authentication flow
+
+    ```javascript
+    clientId: 'GHtf9iJdr60A9IYrR0jw'
+    ```
+Optional configurations :
+
+- **parseSchema:** Callback used to mold the JSON schema that comes back from Okta API.
+
+    ```javascript
+    /**
+     The callback function is passed 3 arguments:
+     @params : schema, onSuccess, onFailure
+     1) @param {Object} schema: json schema returned from the API.
```

Comment:
We do not use JSDoc anywhere else in this `README`. I'd recommend either removing the comment (since we include an example already) or bring the block comment inside the callback, as shown in [#username-and-password](https://github.com/nikhilvenkatraman-okta/okta-signin-widget/blob/2ba03e8b559c4613753f6713cfec267e20fe1e07/README.md#username-and-password)

Reply:
Same as in the **preSubmit** and **postSubmit** functions

Reply:
Removed jsDoc style comments and multi line comments